thorlcr/activities/keyedjoin/thkeyedjoin.cpp

```diff
@@ -27,7 +27,7 @@
 #include "thkeyedjoin.ipp"
 #include "jhtree.hpp"
 
-static const std::array<StatisticKind, 8> progressKinds = {{ StNumIndexSeeks, StNumIndexScans, StNumIndexAccepted, StNumPostFiltered, StNumPreFiltered, StNumDiskSeeks, StNumDiskAccepted, StNumDiskRejected }};
+static const std::array<StatisticKind, 8> progressKinds{ StNumIndexSeeks, StNumIndexScans, StNumIndexAccepted, StNumPostFiltered, StNumPreFiltered, StNumDiskSeeks, StNumDiskAccepted, StNumDiskRejected };
```

Comment:
I think I read that you DO need double {} on c++11 if you are not using = (but not c++14).  See http://en.cppreference.com/w/cpp/container/array